frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -28,14 +30,15 @@ const FormItem = Form.Item;
 type OwnProps = {|
   datastores: Array<APIDataStore>,
   withoutCard?: boolean,
-  onUploaded: (string, string, boolean, ?Vector3) => Promise<void>,
+  onUploaded?: (string, string, boolean, ?Vector3) => Promise<void>,
```

Comment:
I made this argument optional because of the previous (and still existing) inconsistency on what happens after uploading a dataset.
I would prefer to somehow make the behaviour consistent for the "normal upload" and the onboarding.

Reply:
We already discussed a solution for this :rocket: